market_maker/bitmex.py

```diff
@@ -105,9 +105,9 @@ def wrapped(self, *args, **kwargs):
         return wrapped
 
     @authentication_required
-    def funds(self):
+    def funds(self, currency):
```

Comment:
Nit: It would be better if we could default this second argument to maintain backwards compatibility

Reply:
Thanks. Addressed in dc75d1846a9661841e97fd65860a725d6e948dda